Beams/MRML/vtkMRMLRTPlanNode.h

```diff
@@ -95,6 +95,9 @@ class VTK_SLICER_BEAMS_MODULE_MRML_EXPORT vtkMRMLRTPlanNode : public vtkMRMLNode
 
   /// Add given beam node to plan
   void AddBeam(vtkMRMLRTBeamNode* beam);
+  /// Add given proxy beam node to plan

```

Comment:
Since this is a new way and compared to the existing one more complex, could you please describe here what proxy beam node means so that the other developers know when looking at the new code? Thanks!